Source/SelfService/Web/App.tsx

```diff
@@ -27,51 +27,11 @@ import { TransitionProps } from '@mui/material/transitions';
 import { ThemeScreen } from './screens/themeScreen';
 import '@fontsource/rubik';
 import { ApplicationScreen } from './screens/applicationScreen';
-import { Box } from '@mui/material';
+import { Box, createTheme } from '@mui/material';
 import { ContainerRegistryScreen } from './screens/containerRegistryScreen';
 import { M3ConnectorScreen } from './screens/m3connectorScreen';
+import { themeDark } from './theme/theme';
```

Comment:
Good move to start at the top of the stack for this. What i see already is lingo in code that can be used to communicate with others (than developers) touching the product. I believe, if you keep this in the back of your head when doing the design thinking moving forward,  some creative ideas/solutions could come out of it.